Library/Homebrew/test/download_strategies_spec.rb

```diff
@@ -245,15 +245,15 @@ def setup_git_repo
       let(:url) { "https://example.com/foo.tar.gz/from/this/mirror" }
 
       it {
-        is_expected.to eq(
+        expect(subject).to eq(
```

Comment:
This also seems wrong.

Reply:
@reitermarkus RuboCop RSpec doesn't like `is_expected`.

Reply:
Why not?

Reply:
It said uses of the subject should not be implicit.

Reply:
I think this is only because it is not on a single line: https://rubocop-rspec.readthedocs.io/en/latest/cops_rspec/#rspecimplicitsubject

Reply:
@reitermarkus Cool, apologies for applying the autofix unthinkingly then. Happy to see it changed back 👍